WooCommerce/UITestsFoundation/Screens/Products/ProductsScreen.swift

```diff
@@ -17,14 +18,9 @@ public class ProductsScreen: ScreenObject {
         (try? ProductsScreen().isLoaded) ?? false
     }
 
-    init(app: XCUIApplication = XCUIApplication()) throws {
+    public init(app: XCUIApplication = XCUIApplication()) throws {
         try super.init(
-            expectedElementGetters: [
-                topBannerCollapseButtonGetter,
-                topBannerInfoLabelGetter,
-                // swiftlint:skip:next opening_brace
-                { $0.buttons["product-search-button"] }
-            ],
+            expectedElementGetters: [ { $0.buttons["product-add-button"] }, { $0.buttons["product-scan-button"] }, { $0.buttons["product-search-button"]} ],
```

Comment:
Just to make it fit the laptop screens:

```suggestion
            expectedElementGetters: [
                { $0.buttons["product-add-button"] },
                { $0.buttons["product-scan-button"] },
                { $0.buttons["product-search-button"] }
            ],
```

Reply:
Yeah the reason I had it in one line was to avoid adding the lint exclusion comment, I'll see if I can apply it to the whole file instead of per line.